services/ui-src/src/components/reports/sections/StaticDrawerSection.tsx

```diff
@@ -38,6 +38,13 @@ export const StaticDrawerSection = ({ form, page, loadingState }: Props) => {
   const { message, link } =
     emptyVerbiage[entityType as keyof typeof emptyVerbiage];
 
+  // shape entity data for hydration
+  const formData = {
+    fieldData: {
+      ...currentEntity,
+    },
+  };
```

Comment:
Nit: `currentEntity` doesn't need to be spread, so this could just be: 

```
```suggestion
  const formData = { fieldData: currentEntity };
```

Being as it's so short, we could probably even just pass it on line 120 straightaway:


```
formData={ fieldData: currentEntity }
```